src/settings_translation_file.cpp

```diff
@@ -3,14 +3,13 @@
 // To update it, refer to the bottom of builtin/mainmenu/dlg_settings_advanced.lua
```

Comment:
unrelated modification

Reply:
nah, it's okay to regenerate this
just don't squash it on merging